tools/test-framework/src/relayer/channel.rs

```diff
@@ -431,6 +431,8 @@ pub fn assert_eventually_channel_upgrade_ack<ChainA: ChainHandle, ChainB: ChainH
     handle_b: &ChainB,
     channel_id_a: &TaggedChannelIdRef<ChainA, ChainB>,
     port_id_a: &TaggedPortIdRef<ChainA, ChainB>,
+    channel_state_a: ChannelState,
+    channel_state_b: ChannelState,
```

Comment:
I added these extra parameters so that I could test different channel end states on ACK.